### PR TITLE
Decompile func_802400BC_A4C01C

### PR DESCRIPTION
**Pull request by stephenJohnGiardina**

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀🚀🚀🚀🚀🚀🚀🚀🚀 This PR matches 9 functions (+0.45%) on `us`.

⚠️ This PR introduces a warning:
- src/16F740.c:2051: warning: assignment makes pointer from integer without a cast
- src/16F740.c:210: warning: assignment makes pointer from integer without a cast
- src/16F740.c:213: warning: assignment from incompatible pointer type
- src/16F740.c:4316: warning: assignment makes pointer from integer without a cast
- src/1a1f0_len_5390.c:259: warning: implicit declaration of function `spawn_drops'
- src/1a1f0_len_5390.c:671: warning: implicit declaration of function `partner_enable_input'

**Review by nanaian — APPROVED** [review]

File: src/world/area_kpa/kpa_04/A4BF60.c [hunk 1]
```diff
@@ -5,4 +5,30 @@ static char* N(exit_str_1) = "kpa_81";
 
 #include "world/common/UnkFunc11.inc.c"
 
-INCLUDE_ASM(s32, "world/area_kpa/kpa_04/A4BF60", func_802400BC_A4C01C);
+ApiStatus func_802400BC_A4C01C(Evt *script, s32 isInitialCall) {
+    PlayerStatus* playerstatus = &gPlayerStatus;
+    f32 sp24;
```
Comment (nanaian) [inline src/world/area_kpa/kpa_04/A4BF60.c:10]: Name this variable `dist`, see `npc_raycast_down_sides`'s signature for more info

**Review by nanaian — COMMENTED** [review]
Great first match!

**Comment by ethteck** [comment]
nice job!

**Comment by ethteck** [comment]
ah, whoops. @stephenJohnGiardina you targeted the misc-decomp branch in this repo instead of master - I'm gonna revert this and let's re-target it to master

**Comment by ethteck** [comment]
oh, I tried to create a new PR but it looks like you based your misc-decomp branch off of mine (the one in the main repo). It's probably better to just not use the name misc-decomp in your personal fork to avoid the hassle of having to explicitly not track the upstream branch of the same name. 

For now, the easiest thing is to just create a new branch off of master and re-apply the decompilation to that, and make a new PR. let us know if you have any questions

**Pull request merged**